verification/curator-service/ui/cypress/support/commands.ts

```diff
@@ -29,6 +29,7 @@ declare global {
 }
 
 export function addCase(opts: {
+    uploadId?: string;
```

Comment:
nit, but optional params should be after non-optional params (and this part might need to be merged)

Reply:
Done